### PR TITLE
Update backup-and-restore-tool.md

### PR DESCRIPTION
**Pull request by kissmydb**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: br/backup-and-restore-tool.md [hunk 1]
```diff
@@ -64,6 +54,20 @@ SST 文件以 `storeID_regionID_regionEpoch_keyHash_cf` 的格式命名。格式
 - keyHash：Range startKey 的 Hash (sha256) 值，确保唯一性
 - cf：RocksDB 的 ColumnFamily（默认为 `default` 或 `write`）
 
+## 使用方式
+
+目前支持两种方式来运行备份恢复，分别是 SQL 和 命令行工具。
+
+### 通过 SQL
+
+在 v4.0.3 及以上版本的 TiDB 中，已经可以支持通过 SQL 语句进行备份恢复，详见 [Backup 语法](/sql-statements/sql-statement-backup.md#backup) 以及 [Restore 语法](/sql-statements/sql-statement-restore.md#restore)
```
Comment (3pointer) [inline br/backup-and-restore-tool.md:63]: 4.0.2

File: br/backup-and-restore-tool.md [hunk 2]
```diff
@@ -64,6 +54,20 @@ SST 文件以 `storeID_regionID_regionEpoch_keyHash_cf` 的格式命名。格式
 - keyHash：Range startKey 的 Hash (sha256) 值，确保唯一性
 - cf：RocksDB 的 ColumnFamily（默认为 `default` 或 `write`）
 
+## 使用方式
+
+目前支持两种方式来运行备份恢复，分别是 SQL 和 命令行工具。
```
Comment (TomShawn) [inline br/backup-and-restore-tool.md:59]:
```suggestion
目前支持两种方式来运行备份恢复，分别是 SQL 和命令行工具。
```

**Comment by TomShawn** [comment]
@3pointer PTAL again

**Pull request merged**